packages/flutter/lib/src/widgets/text.dart

```diff
@@ -210,16 +215,40 @@ class Text extends StatelessWidget {
     this.textScaleFactor,
     this.maxLines,
   }) : assert(data != null),
+       span = null,
        super(key: key);
 
+  /// Creates a text widget from a [TextSpan].
+  const Text.fromSpan(this.span, {
```

Comment:
"from" doesn't feel right here. It's not "from" a TextSpan any more than the other constructor is "from" a String. It's just that the child is a TextSpan.

Maybe `Text.rich`, following the RichText pattern?

Reply:
Done